packages/cli/test/unit/commands/project/add.test.ts

```diff
@@ -25,5 +25,31 @@ describe('add', () => {
         `Success! Project test-project added (${user.username})`
       );
     });
+
+    it('tracks argument', async () => {
+      useUser();
+      useProject({
+        ...defaultProject,
+        id: 'test-project',
+        name: 'test-project',
+      });
+
+      client.setArgv('project', 'add', 'test-project');
+      await projects(client);
+
+      const project: Project = await client.fetch(`/v8/projects/test-project`);
```

Comment:
I know this was copied from above but... what is it doing?

Reply:
Good catch! Looks like it's just testing the `useProject` mock. I'll make a followup PR to remove this